CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+    - Issue 127 - Provide a better error message when there is no data in the star/end dates requested
```

Comment:
Can you move this to changed and indicate we're switching from 404 to 400 response code

Reply:
Moved to changed and updated issue description.